presto-docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -211,6 +213,8 @@ Limitations
 * Dynamic filtering is currently implemented only for :doc:`/connector/hive` and :doc:`/connector/memory` connectors.
 * Push down of dynamic filters into local table scan on worker nodes is limited to broadcast joins.
 * Min-max dynamic filter collection is not supported for DOUBLE, REAL and unorderable data types.
+* When ``IS NOT DISTINCT FROM`` is used in a join condition, dynamic filtering is not
+  supported if DOUBLE or REAL data types are part of the comparison condition.
```

Comment:
`are part of the comparison condition` -> `are used in the comparison expression`

Reply:
quote DOUBLE and REAL if you uppercase them